### ISSUE TITLE
Crash in new workspace in 0.5

### ISSUE THREAD

**Issue by jonathanBieler** [JuliaGraphics/Gtk.jl#268]
Doing

    using Gtk
    workspace()
    using Gtk

Crashes in 0.5. It works in 0.4. Tested on OSX and Win7.

Here's the log:

https://gist.github.com/jonathanBieler/cff72ede2f18ced8a4eef99ae1d9a810

**Comment by tknopp** [comment]
Thats beyond my knowledge. Ping @vtjnash

**Comment by lobingera** [comment]
@tknopp Can you reproduce? I can.

**Comment by tknopp** [comment]
yes I can. But still this requires someone with knowledge on precompilation.
